src/Lumi/Styles/Box.purs

```diff
@@ -16,7 +16,7 @@ box =
         , flexDirection: column
         , boxSizing: borderBox
         , minHeight: px 0
-        , minWidth: minContent
+        , minWidth: str "auto"
```

Comment:
We'll need to install this version in a staging environment at many screen sizes, since this could potentially cause layout changes.

(btw, there's an `auto` import if you want, but not a big deal)

Reply:
Oh nice I'll use that import, and yep going to get this set up on a staging env shortly (this one def requires robust QA, but I've been running it locally on all our apps since Monday and as I've been working on other things/continually checking on how pages have been impacted by this change, thus far everything seems ok 🤞).

Reply:
Ohh, great! 🎉